app/base-env/llvm.sh

```diff
@@ -10,8 +10,21 @@
 
 set -eux
 
+usage() {
```

Comment:
Why not retrieve this file from https://apt.llvm.org/llvm.sh ?

Reply:
That is another option. Either way I would have to compute the sha. Copying the file is roughly the same overhead.